packages/synapse-react-client/.storybook/main.ts

```diff
@@ -10,15 +10,16 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
     {
-      name: '@storybook/addon-styling',
+      name: '@storybook/addon-themes',
       options: {},
     },
-    'storybook-addon-designs',
+    '@storybook/addon-designs',
   ],
 
   framework: {
     name: '@storybook/react-vite',
     options: {
+      strictMode: false,
```

Comment:
I considered enabling StrictMode by default, but wasn't sure if that might cause problems with stories. Explicitly setting it here makes it easy to go in and change it locally when desired. In the future, we can decide to enable it globally.